include/opt-sched/Scheduler/enumerator.h

```diff
@@ -1387,9 +1400,9 @@ inline int Enumerator::GetSearchCnt() { return iterNum_; }
 
 inline void Enumerator::CreateNewRdyLst_() {
   ReadyList *oldLst = rdyLst_;
-
-  rdyLst_ = new ReadyList(dataDepGraph_, prirts_, SolverID_);
-
+  //Logger::Info("creating new ready list");
```

Comment:
Please remove all commented code before merging. There is already an unfortunate amount in the codebase and we should at least not be adding to it.